src/app/components/AdvocatesTable.tsx

```diff
@@ -0,0 +1,49 @@
+"use client";
+
+import { Advocate } from "@/app/types";
+
+type AdvocatesTableProps = {
+  advocates: Advocate[];
+};
+
+const AdvocatesTable: React.FC<AdvocatesTableProps> = ({ advocates }) => (
+  <div className="overflow-x-auto">
+    <table className="min-w-full bg-white shadow-md rounded-lg overflow-hidden">
+      <thead className="bg-blue-600 text-white">
+        <tr>
```

Comment:
Fixes: #3 Table Elements Not Inside 

By ensuring `<tr>` is inside of `<thead>` we ensure consistent styling and semantic correctness. This also addressed a bug where the row styling was off.